lab-jinho/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fs = require('fs');
+
+fs.readFile(`${__dirname}/data/one.txt`, function(err, data) {
+  if (err) throw err;
+  console.log('content:', data.toString('hex', 0, 8));
+
+  fs.readFile(`${__dirname}/data/two.txt`, function(err, data) {
+    if (err) throw err;
+    console.log('content:', data.toString('hex', 0, 8));
+
+    fs.readFile(`${__dirname}/data/three.txt`, function(err, data) {
+      if (err) throw err;
+      console.log('content:', data.toString('hex', 0, 8));
+    });
+  });
+});
```

Comment:
Good job making use of nested callbacks to meet the needs of the assignment, luckily you won't have to do any more nested ridiculousness after this assignment!